namespaces/ovirt.yml

```diff
@@ -0,0 +1,27 @@
+
+namespace:
+  name: ovirt
+  type: group
+  description: >
+    Namespace for ovirt metadata.
+  fields:
+  - name: entity
+    type: string
+    description: >
+      The type of the data source, hosts, vms, engine etc.
+
+  - name: host_id
+    type: string
+    description: >
+      The oVirt host UUID.
+
+  - name: engine
+    type: group
```

Comment:
If we add new field that is related to oVirt engine I would like to have it under "engine" name space.

Just like we nest the fieldד under collectd.
We could have just have fields names that have dots between each part but we decided to nest them so if theט are under the same plugin for example they will be nested all under that plugin name.